packages/seedless-onboarding-controller/src/SeedlessOnboardingController.ts

```diff
@@ -231,6 +243,8 @@ export class SeedlessOnboardingController<EncryptionKey> extends BaseController<
     socialLoginEmail?: string;
     refreshToken?: string;
     revokeToken?: string;
+    accessToken?: string;
+    metadataAccessToken?: string;
```

Comment:
why is it optional here but then in many/most/all? places we require this to be defined?

Reply:
I also wondered the same but this is not something i added in this PR, same is being done for other tokens because authenticate function is used at two different places and at one of the place these tokens are not required to be passed.  Not the best way to do it, we should have defined separate functions for that.

Reply:
this is a breaking change then.
how would this change be reflected in the frontend code? where would the new tokens be retrieved from?

it is usually required to create a draft PR in at least one of the frontend repositories (extension/mobile) to ensure a proper integration is possible.

Reply:
Nothing will break even if you don't pass these tokens, so its technically not a breaking change.